templates/engage/financial-services-month2.html

```diff
@@ -0,0 +1,181 @@
+{% extends "engage/base_engage.html" %}
+{% block meta_description %}The fintech infrastructure is moving on. A series of webinars to have an understanding of the emerging technologies: multi-cloud, AI/ML, Blockchain and Containers, and their impact on the financial services industry.{% endblock %}
+{% block title %}Canonical presents Financial Services Month{% endblock %}
+{% block content %}
+<section class="p-strip--dark is-deep p-takeover--financial-services">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-7">
+      <h1 class="p-takeover__title">Canonical presents<br> Financial Services month</h1>
+      <p>Filmed at the London Stock exchange with representatives from Canonical, 451 Research, IBM, and a former Deutsche Bank storage CTO, Canonical took a deep dive look in to the rise of multi-cloud in the financial services industry.</p>
+      <p>Join us for insights around emerging technologies including AI/ML, Blockchain and Containers in FinTech and contact us to discuss your business case.</p>
+    </div>
+    <div class="col-1">&nbsp;</div>
+    <div class="col-5 u-align--center u-vertically-center">
```

Comment:
You can add the class "prefix-1" here to have an empty col-1 space to the left (or you could add suffix-1 to the col-7). It's not crazy important here, but generally we should try to maintain a 12 column grid.